content/F2014/F14-media-sarah-sharp.md

```diff
@@ -219,30 +219,29 @@ for that because I negotiated with the professors so I could get credit.
 
 The lesson I took was basically rules can be bent.  Being able to talk with
 your professors and being able to get extra credit for cool projects or do
-projects with profs, that wasn't something I really thought of as this high
```

Comment:
I'm pretty sure I grabbed "profs" from the actual talk recording, not sure how verbatim you want this to be.